maintnotifications/handoff_worker.go

```diff
@@ -506,7 +506,7 @@ func (hwm *handoffWorkerManager) closeConnFromRequest(ctx context.Context, reque
 			internal.Logger.Printf(ctx, "redis: failed to close connection: %v", err)
 		}
 		if internal.LogLevel.WarnOrAbove() {
-			internal.Logger.Printf(ctx, logs.NoPoolProvidedCannotRemove(conn.GetID(), err))
```

Comment:
lets keep the `err` but support `nil` when logging.

Reply:
The error it might log is about the closed connection ?

Shouldn't we log the warning only when there is no error?

I mean the code could return on error, and then the error is not passed to the NoPoolProvidedCannotRemove

I can do the change you request, but I would like to challenge you on the need to log something that sounds unrelated to the warning message ?

I might be wrong. I don't mind 😁

Just tell me.